packages/design-system/src/components/VerticalNav/VerticalNav.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import classNames from 'classnames';
+import VerticalNav from './VerticalNav';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof VerticalNav> = {
+  title: 'Components/Vertical Nav',
+  component: VerticalNav,
+  args: {
+    collapsed: false,
+    nested: false,
+  },
+};
+export default meta;
+
+type Story = StoryObj<typeof VerticalNav>;
+
+const Link = ({ className, ...props }) => (
+  // <Link to={props.href} {...props}>{props.children}</Link>
```

Comment:
Does this commented line need to be here?

Reply:
No, good 👁️ !